app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :user_id, presence: true
+  validates :post_id, presence: true
+end
```

Comment:
- [ ] Let's add `update_likes_counter`. Make it private and use `after_save` callback as described above.